src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/ReportsController.java

```diff
@@ -88,11 +75,26 @@ public ResponseEntity downloadZipFilesSummary(
     ) throws IOException {
         List<ZipFileSummaryResponse> summary = this.reportsService.getZipFilesSummary(date, jurisdiction);
 
-        File csvFile = CsvWriter.writeZipFilesSummaryToCsv(summary);
+        File csvFile = CsvWriter.writeZipFilesSummaryToCsv(toZipFileSummaryResponse(summary));
         return ResponseEntity
             .ok()
             .header(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=zip-files-summary.csv")
             .body(Files.readAllBytes(csvFile.toPath()));
     }
 
+    private List<ZipFilesSummaryReportItem> toZipFileSummaryResponse(List<ZipFileSummaryResponse> summary) {
```

Comment:
function name inconsisten with the actual return type. Maybe just wrap in `ZipFilesSummaryReportListResponse` below.

Reply:
or, even better, make this PR about milliseconds only and leave it as it was

Reply:
sounds better. will update the pr